components/net/cookie.rs

```diff
@@ -94,10 +98,25 @@ impl Cookie {
             return None;
         }
 
+        // https://tools.ietf.org/html/draft-west-cookie-prefixes-04#section-4
+        // Step 1 of cookie prefixes
+        if (cookie.name().starts_with("$Secure-") || cookie.name().starts_with("$Host-")) &&
```

Comment:
Looking at the spec, I see that the more recent version has changed from $prefix to __prefix (https://tools.ietf.org/html/draft-ietf-httpbis-cookie-prefixes-00#section-5.2) -- is there a reason we need to use $ instead of __?

Reply:
Oh, I was looking at the old versions of the spec at https://tools.ietf.org/html/draft-west-cookie-prefixes-04. I should probably rewrite some of this code in order to conform to the latest spec.